### PR TITLE
:lipstick: update UI to bdxio

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+![Screencapture](https://i.goopics.net/01nxw.png)
 
 ## Sources
  - [reactive programming -medium- @tkssharma](https://medium.com/@tkssharma/reactive-programming-rx-js-introduction-32bf963eee1b)
  - [xstream README.md](https://github.com/staltz/xstream)
  - [cycle.js.org documentation](https://cycle.js.org/getting-started.html)
 
 ## Slides
- - [Web2day 2017](https://docs.google.com/presentation/d/1ffUXgs4dimSTOKi3RtfW_6FYeabH9mQs19S1Om2-73k/edit?usp=sharing)
+ - [BDX.io 2017](https://docs.google.com/presentation/d/1QipKxyUIcntcfenZPb6IzpMv11nKSCUyfVfB5VIgaWI/edit?usp=sharing)
```
Comment (fabienjuif) [inline README.md:10]: Let the old version, add the new one in a new line

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # Cycle Music
-![Screencapture](https://pbs.twimg.com/media/DBAOyg2XoAAUDjl.jpg:large)
+![Screencapture](https://i.goopics.net/01nxw.png)
```
Comment (fabienjuif) [inline README.md:2]: Maybe we should capture the application without the event name

File: src/components/marketing/logo/logo.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import xs from 'xstream'
 import { img } from '@cycle/dom'
 
 const view = () => xs.of(
```
Comment (fabienjuif) [inline src/components/marketing/logo/logo.js:4]: We should make master without event logo.
Then do a branch by event

**Review by fabienjuif — CHANGES_REQUESTED** [review]
Make the `master` branch a white application.
Make branch for `Web2day` and an other one for `BDX.io`

File: README.md (hunk 1 above)
Comment (guillaumecrespel) [reply README.md:10]: if i make one branch by event ? i let the old version ?

File: README.md (hunk 2 above)
Comment (guillaumecrespel) [reply README.md:2]: yes, for master only ?

Comment (fabienjuif) [reply README.md:2]: Yes

File: src/components/marketing/logo/logo.js (hunk 3 above)
Comment (guillaumecrespel) [reply src/components/marketing/logo/logo.js]: i wanted to put alakarte logo, waht do you think ?

Comment (guillaumecrespel) [reply src/components/marketing/logo/logo.js:4]: :ok_hand:

File: README.md (hunk 2 above)
Comment (guillaumecrespel) [reply README.md:2]: :ok_hand:

**Pull request closed without merging**